src/ng/parse.js

```diff
@@ -1703,11 +1703,11 @@ function $ParseProvider() {
 
   this.$get = ['$filter', '$sniffer', function($filter, $sniffer) {
     var $parseOptions = {
-          csp: $sniffer.csp,
+          csp: $sniffer.csp && $sniffer.csp.noUnsafeEval,
```

Comment:
Looking at [$sniffer](https://github.com/petebacondarwin/angular.js/blob/ae2241522611cebc8d0833f0bed58bb726c6fdb1/src/ng/sniffer.js#L81), I think the check if  `$sniffer.csp ` exists is unnecessary.